galata/src/helpers/filebrowser.ts

```diff
@@ -217,12 +217,12 @@ export class FileBrowserHelper {
       );
 
     // wait for network response or timeout
-    await this.contents.waitForAPIResponse(
-      async () => {
+    await Promise.race([
```

Comment:
Since we cannot easily test jupyterlite as downstream, is it worth adding a comment explaining why this approach is preferred?

Reply:
Sounds good :+1: